charts/sematext-agent/README.md

```diff
@@ -1,58 +1,25 @@
 # Sematext Agent
 
-Sematext Agent collects logs, metrics, events and more info for hosts (CPU, memory, disk, network, processes, ...), containers and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). Sematext Cloud is available in the US and EU regions.
-
-## Introduction
+Sematext Agent collects logs, metrics, events and more info for hosts (CPU, memory, disk, network, processes, ...), containers (Docker, containerd, cri-o, Podman and rkt) and orchestrator platforms and ships that to [Sematext Cloud](https://sematext.com/cloud). Sematext Cloud is available in the US and EU regions.
 
 This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
 
 ## Prerequisites
 
 - Kubernetes 1.13+
-- You need to create [a new Docker app in Sematext Cloud](https://apps.sematext.com/ui/integrations/create/docker) to get relevant tokens
+- You need to create an Infra App in [Sematext Cloud US](https://apps.sematext.com/ui/monitoring-create/app/infra) or [Sematext Cloud EU](https://apps.eu.sematext.com/ui/monitoring-create/app/infra) to get your Infra App Token.
 
 ## Installation
 
-To install the chart for monitoring and shipping logs run the following command:
-
-```bash
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent \
-    --set infraToken=YOUR_INFRA_TOKEN \
-    --set region=US \
-    sematext/sematext-agent
-```
-
-To install the chart for both logs and monitoring run the following command:
-
-```bash
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent \
-    --set logsToken=YOUR_LOGS_TOKEN \
-    --set infraToken=YOUR_INFRA_TOKEN \
-    --set region=US \
-    sematext/sematext-agent
-```
-
-To provide your infra token as a kubernetes secret instead, create a secret with `infra-token` as a key, and provide its name to the install command:
+To install it, run the following command:
 
-```bash
-$ kubectl create secret generic sematext-secret \
-    --from-literal=infra-token=YOUR_INFRA_TOKEN \
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent \
-    --set existingSecret.name=sematext-secret \
-    --set existingSecret.hasInfraToken=true \
-    --set region=US \
-    sematext/sematext-agent
+```sh
+helm install --name sematext-agent \
+  --set infraToken=<YOUR_INFRA_TOKEN> \
+  --set region=<"US" or "EU"> \
```

Comment:
No quotes needed from what I can tell - https://sematext.com/docs/agents/sematext-agent/kubernetes/helm/

Reply:
I removed them so it's the same info on both. It works without them